bin/init-lts-line

```diff
@@ -11,22 +11,22 @@ baseline=$1
 
 # Make tagged commit part of the release branch
 branch_name="stable-${baseline}"
-git checkout -b $branch_name jenkins-${baseline}
+# Use reincrementalification commit as branch point: second child of release commit (first is `[maven-release-plugin] prepare for next development iteration`)
+git checkout -b $branch_name $(git log --reverse --ancestry-path --pretty=%H jenkins-${baseline}..master | head -2 | tail -1)
```

Comment:
The most recent example: https://github.com/jenkinsci/jenkins/commit/bc37379c96daef081c2ed0fa7ed85a06eadfaa80

Reply:
This needs to be amended as of https://github.com/jenkinsci/jenkins/pull/3567 to just be `| head -1` (i.e., first child of release commit).